heudiconv/convert.py

```diff
@@ -611,6 +619,20 @@ def save_converted_files(res, item_dicoms, bids_options, outtype, prefix, outnam
                             imgtype, "_echo-%d%s" % (echo_number, imgtype)
                         )
                         break
+            # Now check if this run is multi-orientation (v-nav or localizer)
+            if bids_meta and is_multiorient and 'acq-' not in this_prefix_basename:
```

Comment:
I think that this condition should not check if there is `_acq-` here but rather below right before modifying the actual prefix -- announcing that because _acq- is there we would not be embedding multi-orientation information which would have been such as estimated.